cmd/age/age.1

```diff
@@ -0,0 +1,162 @@
+.Dd June 28, 2020
+.Dt AGE 1
+.Sh NAME
+.Nm age
+.Nd a simple, modern and secure file encryption tool
+.Sh SYNOPSIS
+.Nm
+.Fl r Ar recipient
+.Op Fl a
+.Op Fl o Ar output
+.Op Ar input
+.Nm
+.Fl -decrypt
+.Op Fl i Ar key
+.Op Fl o Ar output
+.Op Ar input
+.Sh DESCRIPTION
+In the first synopsis form,
+.Nm
+encrypts
+.Ar input
+to the specified
+.Ar recipient .
+In the second synopsis form,
+.Nm
+decrypts
+.Ar input .
+.Pp
+The following options are available:
+.Bl -tag -width Ds -compact
+.It Fl o , Fl -output Ar output
+Write the result to the file at path
+.Ar output .
+.It Fl a , Fl -armor
+Encrypt to a PEM encoded format.
+.It Fl p , Fl -passphrase
+Encrypt with a passphrase.
+.It Fl r , Fl -recipient Ar recipient
+Encrypt to the specified
+.Ar recipient .
+Can be repeated.
```

Comment:
Does this mean I can do **-r** multiple times for the *same* key (just without getting an error about it) or multiple times to encrypt to *multiple recipients*?

Similar thoughts for **-i**.

This is described later and could probably be omitted here entirely because of that.